Dockerfile

```diff
@@ -1,17 +1,8 @@
-from ubuntu:latest
+from gerbil/gambit
```

Comment:
where is this defined though?

Reply:
This is just the image pushed by gerbil. It's based on previous dockerfile only covering the first half. https://github.com/ober/gerbil-docker/blob/master/official/Dockerfile

Reply:
Pushed the dockerfile, and a makefile for gambit to the etc directory.